xml/Microsoft.Extensions.FileSystemGlobbing.Abstractions/DirectoryInfoWrapper.xml

```diff
@@ -1,319 +1,319 @@
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>Microsoft.Extensions.FileSystemGlobbing.Abstractions.DirectoryInfoBase</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="name" Type="System.String" />
+      </Parameters>
+      <Docs>
+        <param name="name">The directory name.</param>
+        <summary>Returns an instance of <see cref="T:Microsoft.Extensions.FileSystemGlobbing.Abstractions.DirectoryInfoBase" /> that represents a subdirectory.</summary>
+        <returns>The directory.</returns>
+        <remarks>If <paramref name="name" /> equals '..', this returns the parent directory.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="GetFile">
+      <MemberSignature Language="C#" Value="public override Microsoft.Extensions.FileSystemGlobbing.Abstractions.FileInfoBase GetFile (string name);" />
+      <MemberSignature Language="ILAsm" Value=".method public hidebysig virtual instance class Microsoft.Extensions.FileSystemGlobbing.Abstractions.FileInfoBase GetFile(string name) cil managed" />
+      <MemberSignature Language="DocId" Value="M:Microsoft.Extensions.FileSystemGlobbing.Abstractions.DirectoryInfoWrapper.GetFile(System.String)" />
+      <MemberSignature Language="VB.NET" Value="Public Overrides Function GetFile (name As String) As FileInfoBase" />
+      <MemberSignature Language="F#" Value="override this.GetFile : string -&gt; Microsoft.Extensions.FileSystemGlobbing.Abstractions.FileInfoBase" Usage="directoryInfoWrapper.GetFile name" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; override Microsoft::Extensions::FileSystemGlobbing::Abstractions::FileInfoBase ^ GetFile(System::String ^ name);" />
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>Microsoft.Extensions.FileSystemGlobbing</AssemblyName>
+        <AssemblyVersion>1.0.0.0</AssemblyVersion>
+        <AssemblyVersion>1.1.0.0</AssemblyVersion>
+        <AssemblyVersion>2.0.0.0</AssemblyVersion>
+        <AssemblyVersion>2.1.0.0</AssemblyVersion>
+        <AssemblyVersion>2.2.0.0</AssemblyVersion>
+        <AssemblyVersion>3.0.0.0</AssemblyVersion>
+        <AssemblyVersion>3.1.0.0</AssemblyVersion>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+        <AssemblyVersion>7.0.0.0</AssemblyVersion>
+        <AssemblyVersion>8.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>Microsoft.Extensions.FileSystemGlobbing.Abstractions.FileInfoBase</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="name" Type="System.String" />
+      </Parameters>
+      <Docs>
+        <param name="name">To be added.</param>
+        <summary>Returns an instance of <xref data-throw-if-not-resolved="true" uid="Microsoft.Extensions.FileSystemGlobbing.Abstractions.FileInfoBase"></xref> that represents a file in the directory.</summary>

```

Comment:
@carlossanlop This PR also has the weird tags from inheritdoc. I'd rather not bring these into dotnet-api-docs. Do you think they should be brought in?

Reply:
That is weird. Let me address them manually.